daemon/api.go

```diff
@@ -186,6 +187,12 @@ var (
 		POST:   postBuy,
 	}
 
+	readyToBuyCmd = &Command{
+		Path:   "/v2/buy/ready",
```

Comment:
Okay, sounds fine. Let's please just call the same internal method from the final buy endpoint so that we're also doing those verifications when people don't pre-check.